Funky-Chickens/workFORCEmanagement
# Cmjmr editemployee

What was changed:

I created a JSON file to use for seeding the employee computers join table.
I created a seeder file to use this JSON file.

This is because we need some data in this table for testing and fetching later in the app.

TO TEST:
You might want to roll back all previous stuff (sequelize db:migrate:undo:all) to start
Then run sequelize db:migrate
The run sequelize db:seed:all

Run psql in another tab and see if you can view what's in the join table by inputting SELECT * FROM "EmployeeComputers";
